docs/preview/features/correlation.md

```diff
@@ -99,6 +99,16 @@ public class Startup
             // The function that will generate the operation ID header value.
             // (default: new `Guid`).
             options.Operation.GenerateId = () => $"Operation-{Guid.NewGuid()}";
+
+            // Configuration on operation parent ID request header (`Request-Id`).
+            // ------------------------------------------------------------------
+
+            // Whether to extract the operation parent ID from the incoming request (default: false).
```

Comment:
Is there a particular reason we no longer do this? 🤔

Reply:
I would turn it on by default, what do you think @MassimoC @fgheysels ?

Reply:
The reason why this is off is that it enforced the header to be there.

We are changing it to on by default and using best-effort approach. If the header is not there, we rely on the standard operation ID since you don't always have control over the upstream services.